defaults/main.yml

```
@@ -1 +1,20 @@
 clamav_cron_frequency: weekly
+
+# A dictionary of values to be configured in the clamd and freshclam configuration files.
+# Multivalued keys are specified as lists.
+# A Nonetype value means that the key will be removed from the file.
+# Example:
+#
+# clamav_freshclam_conf:
+#   DatabaseMirror: ['db.local.clamav.net', 'database.clamav.net']
+#   Bytecode: 'true'
+#   PrivateMirror:
+# clamv_clamd_conf:
+#   LogFile: /var/log/clamav/clamav.log
+
+
+clamav_freshclam_conf: false
+clamav_clamd_conf: false
```

If these are dictionaries then the default values should be empty dictionaries.

```suggestion
clamav_freshclam_conf: {}
clamav_clamd_conf: {}
```

---

Note that this change will impact the where clauses [here](https://github.com/ClementJ35/ansible-role-clamav/blob/1aadfd5d847e9cca24c6301e819a8f925ec1871c/tasks/main.yml#L62-L74).

---

> Note that this change will impact the where clauses [here](https://github.com/ClementJ35/ansible-role-clamav/blob/1aadfd5d847e9cca24c6301e819a8f925ec1871c/tasks/main.yml#L62-L74).

I tested this change locally and ran into no issues. If there's something I missed please let me know.

---

Ah, OK.  I guess an empty dictionary evaluates to `false` in Ansible.